WeatherApp/Model/WeatherResponse.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  WeatherResponse.swift
+//  WeatherApp
+//
+//  Created on 25/01/2020.
+//  Copyright © 2020 Marcin Wójciak. All rights reserved.
+//
+
+import Foundation
+
+struct WeatherResponse: Decodable {
+    enum CodingKeys: String, CodingKey {
```

Comment:
Jeśli nazwa zmiennej jest różna on nazwy pola z API, to tworzymy `enum` o nazwie `CodingKeys` o 'typie' `String` oraz implementujący protokół `CodingKey`